Bindings/BooleanToVisibilityConverter.cs

```diff
@@ -6,6 +6,10 @@ namespace Bindings
 {
     class BooleanToVisibilityConverter : IValueConverter
```

Comment:
I usually seal the class when adding static default.

Reply:
good point